### PR TITLE
Emphasize that mutiple scopes are allowed in Device flow parameters

### PR DESCRIPTION
**Pull request by 0xTiger**
### Why:
In [Step 1 of the Device Flow Section on the "Authorizing OAuth Apps" page](https://docs.github.com/en/apps/oauth-apps/building-oauth-apps/authorizing-oauth-apps#step-1-app-requests-the-device-and-user-verification-codes-from-github) it's not clear that mutiple scopes can be passed as the description reads "The scope that your app is requesting access to."

Closes: 

### What's being changed (if available, include any code snippets, screenshots, or gifs):
``` markdown
`scope` | `string` | The scope that your app is requesting access to.
```
->
``` markdown
`scope` | `string` | A space-delimited list of [scopes](/apps/oauth-apps/building-oauth-apps/scopes-for-oauth-apps).
```

### Check off the following:

- [x] I have reviewed my changes in staging (look for the "Automatically generated comment" and click the links in the "Preview" column to view your latest changes).
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### CODE REVIEW

**Review by barbarita1975 — COMMENTED** [review]
0xTiger:plural-scopes-allowed

> `[0xTiger:plural-scopes-allowed]() `

**Review by ehsanhajipour5 — COMMENTED** [review]
Vgfgg6

**Review by ehsanhajipour5 — COMMENTED** [review]
Vgfgg6

**Comment by cmwilson21** [comment]
@0xTiger Thanks so much for opening a PR! I'll get this triaged for review :zap:

File: content/apps/oauth-apps/building-oauth-apps/authorizing-oauth-apps.md [hunk 1]
```diff
@@ -145,7 +145,7 @@ The endpoint takes the following input parameters.
 Parameter name | Type | Description
 -----|------|--------------
 `client_id` | `string` | **Required.** The client ID you received from {% data variables.product.product_name %} for your app.
-`scope` | `string` | The scope that your app is requesting access to.
+`scope` | `string` | A space-delimited list of [scopes](/apps/oauth-apps/building-oauth-apps/scopes-for-oauth-apps).
```
Comment (skedwards88) [inline content/apps/oauth-apps/building-oauth-apps/authorizing-oauth-apps.md:148]: minor suggestion to match the style guide:
```suggestion
`scope` | `string` | A space-delimited list of the scopes that your app is requesting access to. For more information, see "[AUTOTITLE](/apps/oauth-apps/building-oauth-apps/scopes-for-oauth-apps)."
```

**Review by skedwards88 — COMMENTED** [review]
Thanks for this improvement! I made one minor suggestion that I will go ahead and incorporate, then we'll get this merged for you.

**Review by skedwards88 — APPROVED** [review]

**Pull request merged**